fflas-ffpack/ffpack/ffpack_ppluq.inl

```diff
@@ -198,8 +198,8 @@ namespace FFPACK {
                     //typename Field::Element * A4R2 = 0;
                     // F = P2 [ L2 ] [ U2 V2 ] Q2
                     //        [ M2 ]
-                    TASK(MODE(CONSTREFERENCE(Fi, P2, Q2, F,/* A4R2,*/ R2) WRITE(R2/*, A4R2[0]*/) READWRITE(F[0], P2, Q2) ),
-                         R2 = pPLUQ( Fi, Diag, M2-R1, N-N2, F, lda, P2, Q2,std::max(nt/2,1))
+                    TASK(MODE(CONSTREFERENCE(Fi, P2, Q2, F,/* A4R2,*/ R2,PSHelper) WRITE(R2/*, A4R2[0]*/) READWRITE(F[0], P2, Q2) ),
+                         R2 = PLUQ( Fi, Diag, M2-R1, N-N2, F, lda, P2, Q2, PSHelper)
```

Comment:
BUG: you should construct a new parseq helper with max(nt/2,1) threads and pass it to this recursive call.